src/app/tests/suites/certification/Test_TC_DISHALM_3_1.yaml

```diff
@@ -11,95 +11,148 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-# Auto-generated scripts for harness use only, please review before automation. The endpoints and cluster names are currently set to default
 
-name: 198.2.2. [TC-DISHALM-3.1] Inflow alarm functionality with DUT as Server
+name: 197.2.2. [TC-DISHALM-3.1] Inflow alarm functionality with DUT as Server
 
 PICS:
     - DISHALM.S
-    - PIXIT.DISHALM.InflowAlarmSupported
+    - DISHALM.S.A0001
+    - DISHALM.S.A0002
+    - DISHALM.S.E00
+    - DISHALM.S.C00.Rsp
+    - PICS_USER_PROMPT
 
 config:
     nodeId: 0x12344321
-    cluster: "Basic Information"
-    endpoint: 0
+    cluster: "Dishwasher Alarm"
+    endpoint: 1
 
 tests:
-    - label: "Step 1: Commission DUT to TH"
-      verification: |
-
-      disabled: true
-
-    - label:
-          "Step 2a: Operate device to set State attribute bit 0 (Inflow) to 1"
-      PICS: DISHALM.M.ManuallyControlled
-      verification: |
-
-      disabled: true
-
-    - label:
-          "Step 2b: After a few seconds, TH reads from the DUT the State
-          attribute"
-      PICS: DISHALM.S.A0002
-      verification: |
-          Bit 0 of response SHALL be 1
-      disabled: true
-
-    - label:
-          "Step 2c: Operate device to set State attribute bit 0 (Inflow) to 0"
-      PICS: DISHALM.M.ManuallyControlled
-      verification: |
-
-      disabled: true
-
-    - label:
-          "Step 2d: After a few seconds, TH reads from the DUT the State
-          attribute"
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Operate device to set State attribute bit 0 (Inflow) to 1"
+      cluster: "LogCommands"
+      command: "UserPrompt"
+      PICS: PICS_USER_PROMPT
+      arguments:
+          values:
+              - name: "message"
+                value: "Please enter 'y' for success"
+              - name: "expectedValue"
+                value: "y"
+
+    - label: "TH reads from the DUT the State attribute"
       PICS: DISHALM.S.A0002
-      verification: |
-          Bit 0 of response SHALL be 0 if bit 0 of DISHALM.S.A0001(Latch) is 0
-          Bit 0 of response SHALL be 1 if bit 0 of DISHALM.S.A0001(Latch) is 1
-      disabled: true
+      command: "readAttribute"
+      attribute: "State"
```

Comment:
This is missing all the Notify event bits from the test plan.  Followup, I guess....